### PR TITLE
Feature/survey import xlsx

### PR DESCRIPTION
**Pull request by OBaddeley**
This PR will convert a xlsx file into a csv format in order to import surveys.

Running the CitizenSpace file through this works, it does awhile

### CODE REVIEW

File: app/services/survey_import_service.rb [hunk 1]
```diff
@@ -54,13 +56,14 @@ def create_survey_question(survey, question)
...
 
-  def create_submission_and_answers(survey) # rubocop:disable Metrics/MethodLength
+  def create_submission_and_answers(survey) # rubocop:disable Metrics/MethodLength,Metrics/AbcSize
     q_a_pairs = []
     submission = nil
     csv.each do |row|
       headers.each do |header|
         answer = row.fetch(header)
         question = survey.survey_questions.find_by(question: header)
+        next if answer.nil?
```
Comment (mermop) [inline app/services/survey_import_service.rb:66]: Could line 65 and line 66 switch place?

File: app/services/survey_import_service.rb [hunk 2]
```diff
@@ -104,8 +107,21 @@ def create_survey_answer(answer, question, submission)
 
   private
 
-  # TODO: convert xlsx files to CSV
   def csv
+    if @file.content_type == Survey::SURVEY_FILE_TYPES[1]
```
Comment (mermop) [inline app/services/survey_import_service.rb:111]: Feel like this is a bit brittle, could be messed up by just reordering `SURVEY_FILE_TYPES`

Comment (OBaddeley) [reply app/services/survey_import_service.rb]: changed 👍

File: app/services/survey_import_service.rb (hunk 1 above)
Comment (OBaddeley) [reply app/services/survey_import_service.rb:65]: done 👍

File: app/workers/survey_import_worker.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class SurveyImportWorker
```
Comment (OBaddeley) [inline app/workers/survey_import_worker.rb:1]: To be used when the front is is built

**Pull request merged**